src/main/java/study/step3/domain/CarRacingController.java

```diff
@@ -0,0 +1,34 @@
+package study.step3.domain;
```

Comment:
패키지 이름으로 step3보다는 racingcar와 같이 도메인 이름을 사용하는 것은 어떨까요?

```suggestion
package study.racingcar.domain;
```